nixos/modules/services/web-apps/moin.nix

```diff
@@ -0,0 +1,273 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  cfg = config.services.moin;
```

Comment:
Oh, by the way, any reason for calling this module `moin` rather than `moinmoin`?

Reply:
I honestly don't know which name is preferred. The project seems to use "MoinMoin" the most, however not lowercase "moinmoin". They also use "moin" for source archive, github repo, CLI etc so I went with that. Happy to change it if matching the package name is more important.

Reply:
From an extremely quick glance at the site it isn't made overly clear which. For consistency sake, though, maybe we should go with `moinmoin`?